data/sql/grid_alternatives.sql

```diff
@@ -138,6 +138,7 @@ VALUES
 -- Spain
 ('100800401.gsb','es_cat_icgc_100800401.tif','100800401.gsb','GTiff','hgridshift',0,NULL,'https://cdn.proj.org/es_cat_icgc_100800401.tif',1,1,NULL),
 ('SPED2ETV2.gsb','es_ign_SPED2ETV2.tif',NULL,'GTiff','hgridshift',0,NULL,'https://cdn.proj.org/es_ign_SPED2ETV2.tif',1,1,NULL),
+('EGM08_REDNAP.asc','es_ign_egm08-rednap.tif','EGM08_REDNAP.asc','GTiff','geoid_like',0,NULL,'https://cdn.proj.org/es_ign_egm08-rednap.tif',1,1,NULL),
```

Comment:
the old_proj_grid_name can be set to NULL as this file has never been released in PROJ data before
```suggestion
('EGM08_REDNAP.asc','es_ign_egm08-rednap.tif',NULL,'GTiff','geoid_like',0,NULL,'https://cdn.proj.org/es_ign_egm08-rednap.tif',1,1,NULL),
```